.phpcs.xml.dist

```diff
@@ -1,10 +1,15 @@
 <?xml version="1.0"?>
 <ruleset name="Jetpack">
 	<config name="minimum_supported_wp_version" value="5.0" />
-	<config name="testVersion" value="5.2-"/>
+	<config name="testVersion" value="5.3-"/>
 
 	<rule ref="PHPCompatibilityWP"/>
-	<rule ref="WordPress-Core" />
+	<rule ref="WordPress-Core">
+		<!-- Can we disable this just for `packages/*`? -->
```

Comment:
I think it may be possible:

```
<!--
    You can hard-code ignore patterns for specific sniffs,
    a feature not available on the command line. Please note that
    all sniff-specific ignore patterns are checked using absolute paths.

    The code here will hide all messages from the Squiz DoubleQuoteUsage
    sniff for files that match either of the two exclude patterns.
 -->
 <rule ref="Squiz.Strings.DoubleQuoteUsage">
    <exclude-pattern>*/tests/*</exclude-pattern>
    <exclude-pattern>*/data/*</exclude-pattern>
 </rule>
```
-- https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-Ruleset